_maps/outpost/hangar/indie_space_20x20.dmm

```diff
@@ -1,79 +1,89 @@
 //MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
 "a" = (
-/turf/open/floor/plating,
+/turf/open/floor/plating{
+	planetary_atmos = 1
+	},
 /area/hangar)
 "b" = (
 /obj/effect/turf_decal/trimline/opaque/yellow/filled/warning{
 	dir = 1
 	},
 /obj/machinery/light/floor/hangar,
-/turf/open/floor/plasteel,
+/turf/open/floor/plasteel{
+	planetary_atmos = 1
+	},
 /area/hangar)
 "e" = (
 /obj/effect/landmark/outpost/hangar_numbers,
-/turf/open/floor/plasteel,
+/turf/open/floor/plasteel{
```

Comment:
Not a big fan of these changes without altering the hangar baseturfs as well. I'm pretty sure that, for instance, once this floor is removed, the plating underneath won't have planetary atmos. More critically, if that *plating* is removed (such as by explosion), the space underneath will suck up all the air created by the other planetary turfs in the hangar, creating a perpetual pressure difference. That seems quite bad.

Reply:
ruh roh

Reply:
i'm not sure I could fix this without making unique subtypes for every type of floor used. shame!!

Reply:
better sooner than later